### ISSUE TITLE
Problem running tests programmatically using Truffle v5.10.0

### ISSUE THREAD

**Issue by calebyouki** [trufflesuite/truffle#6105]
- [x] I've [opened a support ticket](https://trufflesuite.zendesk.com/hc/en-us/requests/new) before filing this issue.

---------------------------

## Issue

When running tests programmatically using the code [here](https://github.com/sc-forks/solidity-coverage/blob/0.7.22/plugins/truffle.plugin.js#L106), the following error is thrown:

```
TypeError:` createInTestDebugFunction is not a function
at Object.<anonymous> (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:295:1)
at Generator.next (<anonymous>)
at /xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:31:1
at new Promise (<anonymous>)
at **webpack_modules**.583725.\_\_awaiter (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:27:1)
at Object.setJSTestGlobals (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:265:1)
at Object.<anonymous> (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:157:1)
at Generator.next (<anonymous>)
at fulfilled (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:28:43)
at runMicrotasks (<anonymous>)
at processTicksAndRejections (node:internal/process/task_queues:96:5)
Truffle v5.10.0 (core: 5.10.0)
Node v16.18.1
```

## Steps to Reproduce

Run the solidity commands in https://github.com/sc-forks/solidity-coverage/blob/0.7.22/plugins/truffle.plugin.js using truffle `v5.10.0`.

## Expected Behavior

Contracts compile successfully and tests are run to completion.

## Actual Results

The following error is thrown:

```
TypeError:` createInTestDebugFunction is not a function
at Object.<anonymous> (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:295:1)
at Generator.next (<anonymous>)
at /xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:31:1
at new Promise (<anonymous>)
at **webpack_modules**.583725.\_\_awaiter (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:27:1)
at Object.setJSTestGlobals (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:265:1)
at Object.<anonymous> (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:157:1)
at Generator.next (<anonymous>)
at fulfilled (/xxxx/node_modules/truffle/build/webpack:/packages/test/dist/src/Test.js:28:43)
at runMicrotasks (<anonymous>)
at processTicksAndRejections (node:internal/process/task_queues:96:5)
Truffle v5.10.0 (core: 5.10.0)
Node v16.18.1
```

## Environment

* Operating System: Linux
* Ethereum client: ganache-core: v2.13.2
* Truffle version (`truffle version`): v5.10.0
* node version (`node --version`): v16.18.1
* npm version (`npm --version`): 8.19.2

**Comment by calebyouki** [comment]
This is a follow-on issue to https://github.com/trufflesuite/truffle/issues/6066

**Comment by eggplantzzz** [comment]
Thanks for reporting! I'll take a look into this.

**Comment by eggplantzzz** [comment]
So it looks like we neglected to handle the case in test when a certain function is not passed. I created a PR [here](https://github.com/trufflesuite/truffle/pull/6118) that should fix this. Thanks again for reporting this @calebyouki!

**Comment by cliffoo** [comment]
Fixed in [v5.10.1](https://github.com/trufflesuite/truffle/releases/tag/v5.10.1). Thanks @calebyouki!

**Comment by calebyouki** [comment]
Thanks @eggplantzzz and @cliffoo for your time and support! It works well now
